Modules/OrgUnit/interfaces/class.ilOrgUnitUserQueriesInterface.php

```diff
@@ -22,4 +22,12 @@ public function findAllUsersByUserIds($user_ids);
 	 */
 	public function getAllUserNames($users);
 
+
+	/**
+	 * @param int $id user id
+	 *
+	 * @return array | null
```

Comment:
Please give a hint about what this array looks like. I hope in some future there will be a Data-Type User which makes clear what to expect, but currently there comes an array and it would help to show an example of such an array in the comment. I see in the implementation of this method you run a `SELECT * ...`. Is this really needed? I don't think so. By defining the array structure here and just define the needed attributes, the implementation then is clear which fields have to be selected.

But beside this: `returnUserDataAsArrayForUserId ` isn't used anyway, or am I wrong? If not needed, just remove it...